js/shared_controls.js

```diff
@@ -686,10 +679,10 @@ function Side(format, terrain, weather, isGravity, isSR, spikes, isReflect, isLi
 }
 
 var gen, genWasChanged, notation, pokedex, setdex, typeChart, moves, abilities, items, STATS, calcHP, calcStat;
-
+/* eslint-disable no-global-assign */
```

Comment:
Ummm after looking it up, this is definitely not a setting that ever needs to be suppressed.

What's the problem here?